openpgp/internal/ecc/ed448.go

```diff
@@ -0,0 +1,102 @@
+// Package ecc implements a generic interface for ECDH, ECDSA, and EdDSA.
+package ecc
+
+import (
+	"crypto/subtle"
+	"io"
+
+	"github.com/ProtonMail/go-crypto/openpgp/errors"
+	ed448lib "github.com/cloudflare/circl/sign/ed448"
+)
+
+type ed448 struct {}
+
+func NewEd448() *ed448 {
+	return &ed448{}
+}
+
+func (c *ed448) GetCurveType() CurveType {
+	return Ed448
+}
+
+func (c *ed448) GetCurveName() string {
+	return "Ed448"
```

Comment:
In ed25519.go, this is lowercase, probably this one should be as well
```suggestion
	return "ed448"
```